.ci/Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM ubuntu:16.04
+
+ENV HADOOP_HOME /usr/local/hadoop
+ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64
+
+# install packages
+RUN \
+  apt-get update && apt-get install -y \
+  ssh \
+  rsync \
+  vim \
+  openjdk-8-jdk
+
+# download and extract hadoop, set JAVA_HOME in hadoop-env.sh, update path
+RUN \
+  wget http://apache.mirrors.tds.net/hadoop/common/hadoop-3.1.2/hadoop-3.1.2.tar.gz && \
+  tar -xzf hadoop-3.1.2.tar.gz && \
+  rm -rf hadoop-3.1.2.tar.gz && \
+  mv hadoop-3.1.2 /usr/local && \
+  ln -sf /usr/local/hadoop-3.1.2/ /usr/local/hadoop
+
+CMD cat hadoop_config >> ~/.bashrc
+
+CMD bash ~/.bashrc
+  ##echo "export JAVA_HOME=$JAVA_HOME" >> $HADOOP_HOME/etc/hadoop/hadoop-env.sh && \
+  ##echo "PATH=$PATH:$HADOOP_HOME/bin" >> ~/.bashrc
+
+##ssh setup
+##RUN \
+##    mkdir ~/.ssh \
+##    chmod 700 ~/.ssh
+
+CMD bash cat "" >> ~/.ssh/authorized_keys \
+    chmod 600 ~/.ssh/authorized_keys
+
+## ssh key gen
+RUN \
+    ssh-keygen -t rsa -P "" -f ~/.ssh/id_rsa
+
+CMD bash cat ~/.ssh/id_rsa.pub >> ~/.ssh/authorized_keys \
+    ssh-copy-id -i ~/.ssh/id_rsa.pub localhost \
+    ssh localhost
```

Comment:
This won't work because the ssh server isn't running. You can safely delete these steps

Reply:
Oh, so there's no need to fetch the authorized keys?

Reply:
Or I should just remove ```ssh localhost``` ?

Reply:
Got it :+1: